uwblueprint/ccbc
# Delete Review Endpoint

## Notion ticket link

[Delete Review](https://www.notion.so/uwblueprintexecs/Review-Service-Delete-Review-db722bf7adbf4b18b85cbe55e1b30cdd)



## Implementation description
* Added endpoint to delete review by ID
* Added test
* Added Swagger docs



## Steps to test
1. Verify that all the tests are passing
2. Add some reviews to DB and try deleting them



## What should reviewers focus on?
* 


## Checklist
